spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -220,6 +220,24 @@
     RUBY
   end
 
+  it 'registers an offense for path with incorrect collapsed namespace' do
```

Comment:
Isn't this the same as the example above?

Reply:
Of course I just missed that 😅 
Let's keep this since it has moar constants 🙂